src/pydvl/reporting/scores.py

```diff
@@ -51,3 +54,49 @@ def compute_removal_score(
         score = u(indices)
         scores[pct] = score
     return scores
+
+
+def compute_removal_score_new(
```

Comment:
As discussed, I'd simply move this function to the notebook support, where it can be adapted as necessary. The old one can be left here and marked for deprecation